platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/util/PropertiesFileSupport.java

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
This seems more like a platform-util class. Is it specific to the admin api somehow?